openfe/tests/protocols/test_openmm_equil_rfe_protocols.py

```diff
@@ -180,10 +216,10 @@ def test_dry_run_complex(benzene_complex_system, toluene_complex_system,
         stateB=toluene_complex_system,
         mapping={'ligand': benzene_to_toluene_mapping},
     )
-    unit = list(dag.protocol_units)[0]
+    dag_unit = list(dag.protocol_units)[0]
```

Comment:
renaming a lot of these because it technically overrides `openff.units.unit` which I've found to be annoying at times (when you try to do further processing later on in the test)